brian_cheung/week_two_challenges/hashes.md

```diff
@@ -91,58 +99,70 @@ groceries = {
   steak: "meaty",
 }
 ```
+```ruby
+groceries = {
+  avocados: "fresh",
+  steak: "meaty",
+  grass: "fiber-filled",
+  whiskey: "smooth",
+  muffins: "sweet",
+  fish: "mercury-tastic",
+  soap: "disgusting"
+}
+```
+
 Programmatically do the following:
 
 1. Avocados are full of that good fat. Change the avocado value to be "fatty".
 
 ```ruby
-# write your code here
+groceries[:avocados] = "fatty"
 ```
 
 2. Muffins are super unhealthy. Change the value for muffins to be "carbtastic".
 
 ```ruby
-# write your code here
+groceries[:muffins] = "carbastic"
 ```
 
 3. Soap? Who would eat soap? Delete that from your hash.
 
 ```ruby
-# write your code here
+groceries.delete(:soap)
 ```
 
 4. The fish went bad. The flavor of fish should be "rotten".
 
 ```ruby
-# write your code here
+groceries[:fish] = "rotten"
 ```
 
 5. Juice cleanse! Add "raw carrot juice" to your hash with the flavor "wallet-draining good".
 
 ```ruby
-# write your code here
+groceries["raw carrot juice"] = "wallet-draining good"
 ```
```

Comment:
Let's stay consistent here - make sure to set this key as a symbol, not a string (and do the same with gluten free bread, below). Symbols should be snake-cased.

Reply:
so like this as output? 
{
...
:"raw carrot juice" => "wallet-draining good"
...
}

Reply:
No, like `:raw_carrot_juice => "wallet-draining good"` or even more like `raw_carrot_juice: "wallet-draining good"`, because that's what the rest of your hash looks like:
```
groceries = {
     avocados: "fresh",
     steak: "meaty",
     grass: "fiber-filled",
     whiskey: "smooth",
     muffins: "sweet",
     fish: "mercury-tastic",
     soap: "disgusting"
}
```
^^The values are strings, but the keys are symbols.